engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -383,11 +383,17 @@ public Source getSource() throws IOException {
    */
   @TruffleBoundary
   public final SourceSection createSection(int sourceStartIndex, int sourceLength) {
-    var src = sources.source();
+    Source src;
+    try {
+      src = getSource();
+    } catch (IOException e) {
+      TruffleLogger logger = TruffleLogger.getLogger(LanguageInfo.ID, Module.class);
```

Comment:
Just use slf4j ... `TruffleLogger` is only good for fast path and this is behind `@TruffleBoundary`.